test/transformers/test_kl_div.py

```diff
@@ -85,12 +83,12 @@ def _test_correctness_once(
 
     output.backward()
     output2.backward()
-    assert_verbose_allclose(x1.grad, x2.grad, atol=atol, rtol=rtol)
+    assert torch.allclose(x1.grad, x2.grad, atol=atol, rtol=rtol)
 
 
 @pytest.mark.parametrize(*_SHAPE_PARAMS)
-@pytest.mark.parametrize("log_target", [False, True])
```

Comment:
these changes seems identical and has no effect?

Reply:
Oh ye, those were just me reordering them to match between both calls because of ocd and probably staged it by mistake.